demo/mkldnn_quant/quant_aware/PaddleCV_mkldnn_quantaware_tutorial.md

```diff
@@ -0,0 +1,221 @@
+# Image classification models fixed-point quantization tutorial
+
+## Overview
+
+Quantization is an important method for model compression and inference performance improvement. PaddleSlim supports two quantization strategies: `post` and `aware`. This tutorial present the way to deploy and inference both post-training models and quantization-aware traing models on CPUS. On Intel (R) Cascade Lake class CPU machines, 8-bits quantization, graph optimizations and DNNL acceleration yields performance of a quantized model up to 4 times better than of an original FP32 model. Currently, we support quantizable operators `conv2d`, `depthwise_conv2d`, `mul`, `matmul`, `transpose2`, `reshape2`, `pool2d`. DNNL optimizations consist mainly of operator fusing passes which simplify the model graph greatly, further improving the performance, including `batch_norm`, `relu`, `brelu`, `elementwise_add`, etc. After quantization and fuses, INT8 models performance will be greatly improved. For details about DNNL optimization users can refer to [SLIM QAT for DNNL INT8](https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/contrib/slim/tests/QAT_mkldnn_int8_readme.md)
+
+**Note**:
+
+- PaddlePaddle in version 1.7.1 or higher is required.
+- DNNL and MKL are required. The highest performance gain can be observed using CPU servers supporting AVX512 instructions.
+- INT8 accuracy is best on CPU servers supporting AVX512 VNNI extension (e.g. CLX class Intel processors). A linux server supports AVX512 VNNI instructions if the output of the command lscpu contains the avx512_vnni entry in the Flags section. AVX512 VNNI support on Windows can be checked using the coreinfo tool.
+
+## 1. Deployment
+
+#### 1.1 Prepare Inference library
+
+Users can build Paddle Inference library from source or directly download from the official website.
+
+- Build Paddle Inference from source, please refer to [Build from source](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/advanced_guide/inference_deployment/inference/build_and_install_lib_cn.html#id12)
+
+| Option        | Value           | Description  |
```

Comment:
Please add some information about what the table is, what are the options (they are CMake options).